src/gov/usgs/traveltime/AuxTtRef.java

```diff
@@ -83,10 +85,10 @@ public AuxTtRef(boolean readStats, boolean readEllip, boolean readTopo)
 		FileLock lock;
 		EllipDeps eDepth;
 		
-		// Set up the properties.
-		if(TauUtil.modelPath == null) {
-			TauUtil.getProperties();
+		if (modelPath != null) {
+			TauUtil.modelPath = modelPath;
```

Comment:
Can this be made non-global.

An example use case is multiple AuxTtRef's processing different model directories from the same process...

Reply:
Not easily, a number of different classes reference the tau util model path

Reply:
Same issue as https://github.com/usgs/neic-locator/pull/21#discussion_r264328716 , using global's makes it difficult to access multiple modelPaths within the same process.  Can those other things using model path receive it as a parameter or similar?

Reply:
My goal with the TT library right now is to keep it functional to support neic-locator and locator evaluation. I would prefer that we make this comment into an issue that can be addressed when I do the code review/standards pass for TT lib after the locator is in testing.